lib/seattleflu/db/cli/command/etl/presence_absence.py

```diff
@@ -0,0 +1,386 @@
+"""
+Process enrollment documents into the relational warehouse
+"""
+import click
+import logging
+import sys
+from datetime import datetime, timezone
+from itertools import groupby
+from operator import itemgetter
+from typing import Any
+from seattleflu.db.session import DatabaseSession
+from seattleflu.db.datatypes import Json
+from . import etl
+
+
+LOG = logging.getLogger(__name__)
+
+
+# This revision number is stored in the processing_log of each enrollment
+# record when the enrollment is successfully processed by this ETL routine.
+# The routine finds new-to-it records to process by looking for enrollments
+# lacking this revision number in their log.  If a change to the ETL routine
+# necessitates re-processing all enrollments, this revision number should be
+# incremented.
```

Comment:
This comment was copied from the enrollment ETL but not updated to refer to presence/absence.